MidasCivil_oM/Settings/MidasCivilAdapterSettings.cs

```diff
@@ -0,0 +1,37 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+
+namespace BH.oM.Adapters.MidasCivil
+{
+    public class MidasCivilAdapterSettings : BHoMObject
```

Comment:
Could I recommend making these an `ActionConfig` and used within the `Push`, `Pull`, and `Execute` components, same as other toolkits/adapters with settings set up for consistency 😄

Reply:
Of course. I looked through the code and found reference in the `Lusas_Toolkit` where the `LusasConfig` is in this place and in the `Revit_Toolkit` I found examples of both. If I understood the code right.

Reply:
@FraserGreenroyd for the version, tolerances, database settings etc. I think this is best set when we run the Adapter rather than at `Push`, `Pull` etc. 

With regards to inheritance, I have noticed quite a few structural adapters use `[SoftwareName]Config`, should we be inheriting `AdapterSettings` instead? I don't think I have seen any toolkits that do that.

Reply:
What's `AdapterSettings`?

`ActionConfig` is the one most toolkits have been implementing to vary things at the push/pull/execute level which these settings appear to be geared up towards doing? Maybe worth a call to discuss?